docs/src/pages/components/text-fields/HelperTextAligned.tsx

```diff
@@ -1,18 +1,16 @@
 import * as React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
 import TextField from '@material-ui/core/TextField';
 
-const useStyles = makeStyles({
-  root: {
-    display: 'flex',
-    alignItems: 'center',
-  },
-});
-
 export default function HelperTextAligned() {
-  const classes = useStyles();
   return (
-    <div className={classes.root}>
+    <Box
+      sx={{
+        '& > :not(style)': {
+          m: 1,
```

Comment:
```suggestion
          display: 'flex',
          alignItems: 'center',,
```
?

Reply:
Before:

![image](https://user-images.githubusercontent.com/57921784/113787088-6978f780-9708-11eb-9126-f112f642b292.png)

After:

![image](https://user-images.githubusercontent.com/57921784/113787119-739af600-9708-11eb-889d-41eaedb13933.png)

For this case `display` and `alignItems` were not relevant, only `margin` was needed.